staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -900,15 +899,16 @@ func toForwardingRuleComposite(rule interface{}) (frc *forwardingRuleComposite,
 	switch fr := rule.(type) {
 	case *compute.ForwardingRule:
 		frc = &forwardingRuleComposite{
```

Comment:
the forwardingRuleComposite struct was added to distinguish between Beta and GA api. Now that we are calling GA api whether or not GlobalAccess is enabled, we can get rid of this struct and use compute.ForwardingRule directly.